docs/connect-to-lms.md

```diff
@@ -13,7 +13,7 @@ For the GitHub Classroom and LMS integration to work, we require that your LMS s
 - LTI version 1.0 and/or 1.1
 - LTI Names and Roles Provisioning 1.X
 
-**Note**: Blackboard and GitHub Classroom course rosters don't work together yet. We are working to support Blackboard in the near future!
+**Note**: Blackboard and Brightspace course rosters don't work together yet with GitHub Classroom. We are working to support Blackboard and Brightspace in the near future!
```

Comment:
This sentence reads a little weird with the edit. Maybe this is a little more clear?

```suggestion
**Note**: Blackboard and Brightspace course rosters don't sync with GitHub Classroom yet. We are working to support Blackboard and Brightspace in the near future!
```

Reply:
Would "aren't compatible" make sense here? "Don't sync" makes it sound like there's a specific syncing behavior that isn't working.

Reply:
I like that change...